docs/reference/user-management.md

```diff
@@ -0,0 +1,107 @@
+2. Enter your details. Your password must be at least eight characters, including at least one number and one capital letter.
+3. Click **Next**. n8n logs you in with your new owner account.
+
+### Step three: invite users
+
+You can now invite other people to your n8n instance. 
+
+1. Sign in with your owner account.
+2. Click your user icon > **Settings**. n8n opens your **Personal settings** page.
+3. Click **Users** to go to the **Users** page.
+4. Click **Invite**.
+5. Enter the new user's email address.
+6. Click **Invite user**. n8n sends an email with a link for the new user to join.
+
+## Manage users
+
+The **Users** page shows all users, including ones with pending invitations. You can:
+
+* Delete a user: 
+  1. Click the menu icon by the user you want to delete.
+  2. Confirm you want to delete them.
+  3. If they are an active user, choose whether to copy their workflow data and credentials to a new user, or permanently delete their workflows and credentials.
+* Resend an invitation to a pending user: click the menu icon by the user, then click **Resend invite**.
+
+## Account types
+
+There are two account types, owner and member. The account type affects the user permissions and access.
+
+* Owner: this is the account that set up user management. There is one owner account for each n8n instance. You can't transfer ownership.  
+  The owner can:
+    * Add and remove users
+    * See all workflows
+    * Delete tags
+* Members: these are normal n8n users.  
+  Members can:
+    * See all workflow tags, create new tags, and assign tags to their workflows. Members can't delete tags.
+    * Change their own password.
+    * See their own workflows.
+
+::: tip Create a member-level account for the owner
+We recommend that owners create a member-level account for themselves. Owners can see all workflows, but there is no way to see who created a particular workflow, so there is a risk of overriding other people's work if you build and edit workflows as an owner.
+:::
+
+## Skipping or disabling user management
+
+You don't have to use n8n's user management feature. You can:
+
+* Leave it enabled, but choose to skip the setup step. You can use n8n as normal. If you want to set up user management later, go to **Settings** > **Users**.
+* Disable the feature completely using the `N8N_USER_MANAGEMENT_DISABLED` environment variable. Setting this environment variable to `true` completely hides the feature in your n8n instance.
+
+## Best practices
+
+This sections contains advice on best practices relating to user management in n8n.
+
+* We recommend that owners create a member-level account for themselves. Owners can see all workflows, but there is no way to see who created a particular workflow, so there is a risk of overriding other people's work if you build and edit workflows as an owner.
+* Multiple users can't edit the same workflow simultaneously. n8n does not support this type of synchronous collaboration.
```

Comment:
You can edit simultaneously. You'll just overwrite each others' changes

Reply:
Changed to:

Users must be careful not to edit the same workflow simultaneously. It is possible to do it, but the users will overwrite each other's changes.